Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -711,6 +728,25 @@ bool EventsRefactorer::SearchStringInConditions(
   return false;
 }
 
+bool EventsRefactorer::GetAllSearchableStrings(gd::ObjectsContainer& project,
+                                               gd::ObjectsContainer& layout,
+                                               gd::EventsList& events,
+                                               gd::String search,
+                                               bool matchCase) {
+  // See later for "BuiltinCommonInstructions::Group" when Comment works
+  if (event.GetType() == = "BuiltinCommonInstructions::Comment") {
+    for (gd::String str : events[id].GetAllSearchableStrings()) {

```

Comment:
Seems like you forgot to iterate on `events`. `events` is an `gd::EventsList`, which, like the name indicates, is a "list", almost an array.

Reply:
Add the missing for, and this can be just:

```
for (gd::String str : event.GetAllSearchableStrings()) {
```

if I suppose you named "event" the "gd::BaseEvent" variable that you will get with "GetEvent(index)" on `events`